app/src/main/java/io/nwhacks/nfc/ReadFragment.java

```diff
@@ -122,18 +122,24 @@ public void tagDiscovered(NFCManager mgr, Intent intent) {
             sb.append(records.get(i));
             sb.append("\n");
         }
+        String applicantCollection;
+        if (records.get(1) != null){
+            applicantCollection = ApplicantInfo.applicantMap.get(records.get(1));
+        }else{
+            applicantCollection = "hacker_short_info";
```

Comment:
Instead of defaulting to `hacker`, we probably want to know when/if this issue occurs because it means that the tag has been tampered with, or the writing of `applicantType` to the tag is not working. Can we toast some sort of error message  like `"Unexpected tag contents, please rewrite applicant's tag."` when this happens and return from the method?

Reply:
Sure, sounds good!